docs/tools/exp_calculator.md

```diff
@@ -0,0 +1,74 @@
+# EXP Calculator
```

Comment:
See other comments related to this, but probably rename to something like:

Stats - Purchasable with EXP